discover/portal/articles/100-web-experience-management/02-building-a-site/03-creating-multiple-navigation-menus/01-default-navigation.markdown

```diff
@@ -0,0 +1,98 @@
+# Default Navigation
```

Comment:
Hey Stephen,

What's the reason for introducing the concept of "Default Navigation". I personally find it very confusing.

Reply:
The idea of "Default Navigation" was to try to differentiate the "default" behavior (create a page and it appears in the hierarchy, drag it around, and it appears differently) from Navigation Menus, which can reorder or reorganize pages any way you like, separate from the "default" organization.

Reply:
Understood. I still believe it's a new concept that it's not necessary, so let's simplify things.

Let's talk about the page hierarchy, which can be used for the site navigation directly and then there are also navigation menus.

Reply:
So from a messaging perspective, we want to have it just be "This is how you create pages. Period." (what I'm calling "default navigation"). And then, separately, "This is a useful way that you can manage and organize pages." (Navigation Menus)?

Reply:
> So from a messaging perspective, we want to have it just be "This is how you create pages. Period." (what I'm calling "default navigation").
Correct

> "This is a useful way that you can manage and organize pages." (Navigation Menus)?

No. Navigation menus are not used to "manage" pages. And it's not correct to think of them as a way to organize pages either.

What we need to say is that navigation menus can be used for sites where additional menus are useful. It's also possible to make the navigation fully based on navigation menus and not use the page hierarchy at all. This will probably make sense only in larger sites with complex navigations.